azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -91,6 +97,7 @@
   private static final String DEFAULT_TIMEZONE_ID = "default.timezone.id";
   private static final int DEFAULT_THREAD_NUMBER = 50;
 
+  private static String hostAlias = null;
```

Comment:
Couple of Questions:
1. Should we let the server run if the obtained `hostAlias` is null or should we crash the server (since metrics and everything will break)?
2. Can `hostAlias` change during execution? Else should it be a `final` variable?